app/src/main/java/org/fossasia/susi/ai/data/db/contract/IDatabaseRepository.kt

```diff
@@ -27,5 +27,5 @@ interface IDatabaseRepository {
     fun closeDatabase()
     fun updateDatabase(prevId: Long, message: String, isDate: Boolean, date: String, timeStamp: String,
```

Comment:
Change this to object

Reply:
@iamareebjamal should I make a pojo class for these parameters? and should I do it in a separate PR, I'll make an issue for it

Reply:
Do it in this PR